help/en/package/jmri/jmrit/etcs/index.shtml

```diff
@@ -0,0 +1,127 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <title>JMRI: ERTMS / ETCS</title>
+  <meta name="author" content="Steve Young">
+  <meta name="keywords" content="JMRI ERTMS ETCS DMI">
+  <!--#include virtual="/help/en/parts/Style.shtml" -->
+</head>
+<body>
+  <!--#include virtual="/help/en/parts/Header.shtml" -->
+
+  <div id="mBody">
+    <div id="mainContent" class="no-sidebar">
+      <h1>JMRI : ERTMS ETCS</h1>
+
+        <p>The European Rail Traffic Management System ( ERTMS ) aims to improve
+        interobability between the multiple systems in use by European railways,
+        making it easier for trains to cross national borders without needing to change systems.</p>
+
+        <p>European Train Control System ( ETCS ) is the signalling component of ERTMS.</p>
+
+
+      <h2>Driver Machine Interface Simulation</h2>
+      
+      <span class="since">since 5.7.4</span>
+      <p>
+      The Driver Machine Interface  ( DMI, the in-cab screen which displays speed and signalling
+      information to the driver ) is one of the most recognisable elements of ETCS.
+      </p>
+
+      <a href="etcs_did_640x478.png">
+      <img src="etcs_did_640x478.png" alt="ERTMS ETCS JMRI DMI" width="320" height="239"></a>
+      
+      <p>The DMI can include information about speed, track conditions, and system alerts.</p>
+
+      <p>It is possible simulate DMI screens and control their content programatically,
```

Comment:
> It is possible `to` simulate DMI screens and control their content `programatically`

I'm not a native English speaker, but I think the word `to` is missing in the sentence after `It is possible`. I have added `to` in my sentence above.

The word `programatically` seems misspelled. Google suggests `programmatically`.